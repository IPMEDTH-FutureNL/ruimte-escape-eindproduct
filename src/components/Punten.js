import React, { Component } from 'react'

export class Punten extends Component {
    constructor(props){
        super(props);
    }
  render() {
    return (
      <div className="punten">Punten: 100</div>
    )
  }
}

export default Punten