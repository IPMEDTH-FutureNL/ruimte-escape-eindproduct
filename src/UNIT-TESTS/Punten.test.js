import { render , screen, fireEvent } from "@testing-library/react"
import {BrowserRouter as Router} from 'react-router-dom';
import Punten from '../components/Punten';

test('on initial render of punten component, check if component is loaded with the correct prop value', () => {
    const { asFragment }  = render(
        <Router>
            <Punten punten="150"></Punten>
        </Router>
    )

    screen.debug();
    expect(asFragment()).toMatchSnapshot();
})
