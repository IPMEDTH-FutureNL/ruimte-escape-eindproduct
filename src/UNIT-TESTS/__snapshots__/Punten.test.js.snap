// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`on initial render of punten component, check if component is loaded with the correct prop value 1`] = `
<DocumentFragment>
  <div
    class="punten"
  >
    Punten: 150
  </div>
</DocumentFragment>
`;
